File: discover_more_python/write_them_to_disc.py
#!/usr/bin/python

import urllib2

# create empty file '20' in tmp
file = open('/tmp/20', 'a')

# create HTTP request
url = 'https://api.github.com/search/repositories?q=language:python&sort=stars&order=desc'
request_headers = {
      'User-Agent': 'Holberton_School',
      'Authorization': 'token a2eaac77d1b665a2f1fea8ce17cd48178cd21a69'
    }
req = urllib2.Request(url, None, request_headers)
response = urllib2.urlopen(req)

# write response to /tmp/20
file.write(response.read())

File: discover_ruby/write_them_to_disc.rb
#!/usr/bin/ruby

require 'httpclient'

filename = 20
filepath = '/tmp/20'
extheaders = {
  'User-Agent' => 'Holberton School',
  'Authorization' => 'token a2eaac77d1b665a2f1fea8ce17cd48178cd21a69'
}

http_request = HTTPClient.new
File.write(filepath, http_request.get_content('https://api.github.com/search/repositories?q=language:ruby&sort=stars&order=desc', query = nil, extheaders))
